pilot/lib/pilot/tester_manager.rb

```diff
@@ -126,8 +127,20 @@ def find_app_tester(email: nil, app: nil)
       return tester
     end
 
+    def find_current_user
+      current_user_email = Spaceship::Tunes.client.user_email
+      current_user_apple_id = Spaceship::Tunes.client.user
+
+      current_user = Spaceship::Members.find(current_user_email)
+      unless current_user
+        UI.user_error!("Unable to find a member for AppleID: #{current_user_apple_id}, email: #{current_user_email}")
+        return nil
```

Comment:
I think this `return` is superfluous because the `user_error!` is going to bail out of this method. ⛵️